boards/nxp/mimxrt700_evk/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+.. _mimxrt700_evk:
+
+NXP MIMXRT700-EVK
+##################
```

Comment:
Thank you for reminding. The board name is mimxrt700_evk, don't like rt500/600 platform.  
Due to board still not released externally, currently I can't update board picture.

Reply:
For Q1 I have update zephyr:board for mimxrt700_evk. Thank you.
For Q2，I can't put picture because the rt700 board don't released externally currently.

Reply:
This hasn't been addressed?

Reply:
Yes, RT700 board won't be released externally recently.

Reply:
I'm sorry, I don't understand. You commented to indicate you have applied "zephyr:board::" directive but you haven't, so I'm confused? It's ok if you don't want to include an image at the moment, but the board documentation page should be correct/complete nevertheless

Reply:
Now  "zephyr:board::" directive had been added into index.rst file. I'm not clear about your mean on the documentation page. What is correct/complete for board documentation page?